tests/foundry/src/Vm2.sol

```diff
@@ -0,0 +1,25 @@
+
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.0 <0.9.0;
+pragma experimental ABIEncoderV2;
+
+import {Vm} from "forge-std/Vm.sol";
+
+interface Vm2 is Vm {
+    // Expects a call using the CALL opcode to an address with the specified calldata.
+    function expectRegularCall(address,bytes calldata) external;
+    // Expects a call using the CALL opcode to an address with the specified msg.value and calldata.
+    function expectRegularCall(address,uint256,bytes calldata) external;
+    // Expects a static call to an address with the specified calldata.
+    function expectStaticCall(address,bytes calldata) external;
+    // Expects a delegate call to an address with the specified calldata.
+    function expectDelegateCall(address,bytes calldata) external;
+    // Expects that no contract calls are made after invoking the cheatcode.
+    function expectNoCall() external;
+    // Expects the given address to deploy a new contract, using the CREATE opcode, with the specified value and bytecode.
+    function expectCreate(address,uint256,bytes calldata) external;
+    // Expects the given address to deploy a new contract, using the CREATE2 opcode, with the specified value and bytecode (appended with a bytes32 salt).
+    function expectCreate2(address,uint256,bytes calldata) external;
+    // Makes the storage of the given address completely symbolic.
+    function symbolicStorage(address) external;
```

Comment:
`symbolicStorage` is added, but not implemented. Should it be added to the unimplimented list?

Reply:
Done. I've added them.